pkg/scheduler/api/resource_info.go

```diff
@@ -40,17 +49,33 @@ type Resource struct {
 	MaxTaskNum int
 }
 
-const (
-	// GPUResourceName need to follow https://github.com/NVIDIA/k8s-device-plugin/blob/66a35b71ac4b5cbfb04714678b548bd77e5ba719/server.go#L20
-	GPUResourceName = "nvidia.com/gpu"
-)
-
 // EmptyResource creates a empty resource object and returns
 func EmptyResource() *Resource {
 	return &Resource{}
 }
 
-// Clone is used to clone a resource type
+// NewResource creates a new resource object from resource list
+func NewResource(rl v1.ResourceList) *Resource {
```

Comment:
If there is no change for the function. We'd better keep it as it is.

Reply:
> If there is no change for the function. We'd better keep it as it is.

No changes about these functions, just resort their orders.